tests/scripts/helpers/kruize.py

```diff
@@ -185,12 +185,17 @@ def list_recommendations(experiment_name=None, latest=None, monitoring_end_time=
 
 # Description: This function deletes the experiment and posts the experiment using createExperiment API to Kruize Autotune
 # Input Parameters: experiment input json
-def delete_experiment(input_json_file, invalid_header=False):
+def delete_experiment(input_json_file, invalid_header=False, rm=True):
     json_file = open(input_json_file, "r")
     input_json = json.loads(json_file.read())
 
     print("\nDeleting the experiment...")
     url = URL + "/createExperiment"
+    if rm:
```

Comment:
As we are relaying on the boolean param which is passed to the function to maintain it's string representation, we can directly use the `string` representation of the boolean instead of placing a `if-else` block.

I was suggesting something like this:
```
PARAMS = {'rm': str(rm).lower()}
```